geowebcache/core/src/main/java/org/geowebcache/config/XMLConfiguration.java

```diff
@@ -431,7 +431,7 @@ public static XStream getConfiguredXStreamWithContext(XStream xs, WebApplication
             xs.allowTypesByWildcard(new String[]{"org.geowebcache.**"});
         }
         
-        xs.setMode(XStream.NO_REFERENCES);
+        xs.setMode(XStream.ID_REFERENCES);
```

Comment:
It makes referencing possible.  It is one of three mechanism's for doing this, which the current setting of NO_REFERENCES stricly disallows.  Of the three it is the easiest to write.